lib/matplotlib/axes/_secondary_axes.py

```diff
@@ -161,9 +158,7 @@ def set_functions(self, functions):
                              'and the second being the inverse')
         self._set_scale()
 
-    # Should be changed to draw(self, renderer) once the deprecation of
-    # renderer=None and of inframe expires.
-    def draw(self, *args, **kwargs):
+    def draw(self, renderer):
```

Comment:
Found this comment. Not sure if this is indeed the correct form or not (deprecation of inframe was removed in 3.5).

Reply:
Seems okay.